docs/UserGuide.md

```diff
@@ -268,26 +287,9 @@ Example:
 * `customise show a/` The application now shows addresses in the list of contacts.
 * `customise show p/ t/` The application now shows phone numbers and tags in the list of contacts.
 
-### Customise Order of Details: `customise order`
-
-Customise the order of information shown in all contacts shown.
-
-Format: `customise order [t/] [p/] [e/] [a/]`
-
-* Name of contact will always be at the top of each contact and cannot have its order changed.
-* Information that can have its order changed: Tags, Phone Number, Email, Address.
-* Information that are not specified will be ordered last and follow the default order. (Tags > Phone Number > Email > Address)
-
-Example:
-
-* `customise order a/ e/ p/` The application will show address first, followed by email, phone number, then tags.
-* `customise order a/` The application will show address first. The rest of the information will follow the default order. Therefore, address will be followed by tags, phone number and then email.
-
-### Clearing all contacts : `clear`
+### Showing all contact details `[coming in v1.5]`
```

Comment:
maybe this should be under `customise show`? Since this feature wouldn't be a standalone feature when implemented, it shouldnt be documented as a standalone feature?

Reply:
Yes it is right under `customise show` but using a smaller header `####` seems too small and inconsistent with the other features.